nav_msgs/Path raw_path
---
nav_msgs/Path smoothed_path